keras_hub/src/models/roformer_v2/roformer_v2_backbone_test.py

```diff
@@ -0,0 +1,40 @@
+import keras
+from keras import ops
+
+from keras_hub.src.models.roformer_v2.roformer_v2_backbone import (
+    RoformerV2Backbone,
+)
+from keras_hub.src.tests.test_case import TestCase
+
+
+class RoformerV2BackboneTest(TestCase):
+    def setUp(self):
+        self.init_kwargs = {
+            "vocabulary_size": 10,
+            "num_layers": 2,
+            "num_heads": 1,
+            "hidden_dim": 2,
+            "intermediate_dim": 4,
+            "head_size": 2,
+        }
+        self.input_data = {
+            "token_ids": ops.ones((2, 5), dtype="int32"),
+            "segment_ids": ops.zeros((2, 5), dtype="int32"),
+        }
+
+    def test_backbone_basics(self):
+        if keras.config.backend() == "torch":
+            import torch
+
+            if torch.cuda.device_count():
+                self.skipTest("Failing on GPU on CI")
+        if keras.__version__ >= "3.6":
```

Comment:
Add this as a skip condition, as above as well (flip the condition and call `self.skipTest`